include/types.h

```diff
@@ -8,4 +8,11 @@ typedef struct Velocity
 {
     // TODO: direction
     float speed;
-} VelocityType;
\ No newline at end of file
+} VelocityType;
+
+typedef struct Scale
+{
+
+    float scale;
+
+} ScaleType;
```

Comment:
Are you planning to have another field in ScaleType? Else may not be worth having a typedef and just use float directly.

Velocity was intended to have both speed and direction but not sure it'll stay either

Reply:
Perhaps, but we could use this to set the size for characters/objects

Reply:
Sure i'm just saying not sure if it's worth making it a struct